src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static Figure getRandomFigure() {
+        Figure[] figures = {
+                new Circle(new Random().nextDouble(), ColorSupplier.getRandomColor()),
+                new IsoscelesTrapezoid(
+                        new Random().nextDouble(),
```

Comment:
Let's make Random an instance of the class and avoid creating multiple instances of Random ;)